cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -0,0 +1,167 @@
+    constrInvToDelete.map((constrInventory) => cy.deleteAwxConstructedInventory(constrInventory));
+    cy.deleteAwxOrganization(organization);
+  });
+
+  it('can create a constructed inventory using specific source_vars and limit and then delete that inventory', () => {
+    invNames = inventoriesList.map(({ name }) => String(name));
+    cy.intercept('POST', awxAPI`/constructed_inventories/`).as('createInv');
+    const constInvName = 'E2E Constructed Inventory ' + randomString(4);
+    // generates random values to be used during the test.
+    const cacheTimeoutValue = generateRandom(0, 15);
+    // which combination should be used to test verbosity from 1 to 5 - UI forces to use only 0-2
+    const verbosityValue = generateRandom(0, 2);
+
+    cy.navigateTo('awx', 'inventories');
+    cy.getByDataCy('create-inventory').click();
+    cy.get('.pf-v5-c-dropdown__menu').within(() => {
+      cy.get('[data-cy="create-constructed-inventory"]').click();
+    });
+    cy.getByDataCy('name').type(constInvName);
+    cy.getByDataCy('description').type(`Description of "${constInvName}" typed by Cypress`);
+    cy.singleSelectBy('[data-cy="organization"]', organization.name);
+    // this can be simplified if we include data-cy to the search button of instance groups
+    cy.getByDataCy('instance-group-select-form-group').find('[aria-label="Options menu"]').click();
+    cy.filterTableByMultiSelect('name', [instanceGroup.name]);
+    cy.getByDataCy('checkbox-column-cell').click();
+    cy.clickModalButton('Confirm');
+
+    cy.multiSelectBy('[data-cy="inventories"]', invNames);
+    cy.getByDataCy('update_cache_timeout').clear().type(String(cacheTimeoutValue));
+    cy.singleSelectByDataCy('verbosity', String(verbosityValue));
+    // FIXME: when verbosity is 0 the filed remains empty
+    cy.getByDataCy('limit').type('5');
+    cy.dataEditorTypeByDataCy('source-vars', 'plugin: constructed');
+    cy.clickButton(/^Create inventory$/);
+    cy.wait('@createInv')
+      .its('response.statusCode')
+      .then((statusCode) => {
+        expect(statusCode).to.be.equal(201);
+        cy.verifyPageTitle(constInvName);
+      });
+    cy.clickKebabAction('actions-dropdown', 'delete-inventory');
+
+    cy.getModal().within(() => {
+      cy.get('header').contains('Permanently delete inventory');
+      cy.get('button').contains('Delete inventory').should('have.attr', 'aria-disabled', 'true');
+      cy.get('[data-cy="name-column-cell"]').should('have.text', constInvName);
+      cy.get('input[id="confirm"]').click();
+      cy.clickButton(/^Delete inventory/);
+    });
+
+    cy.filterTableBySingleSelect('name', constInvName, true);
+  });
+
+  it('can edit and run a sync on the edited constructed inventory', () => {
+    cy.intercept('PATCH', awxAPI`/constructed_inventories/*`).as('saveInv');
+    cy.intercept('POST', awxAPI`/inventory_sources/*/update`).as('syncInv');
+    cy.visit(
```

Comment:
we should not be using hardcoded urls
if the action you are trying to perform is navigating to the details tab, follow this approach

1. `cy.navigate`to the list page
2. search and filter for the resource
3. go to details page
4. `cy.clickTab` of the tab name